test/jdk/javax/swing/JColorChooser/Test4887836.java

```diff
@@ -21,23 +21,43 @@
  * questions.
  */
 
+import java.awt.Color;
+import java.awt.Font;
+import javax.swing.JColorChooser;
+import javax.swing.UIManager;
+
 /*
  * @test
  * @bug 4887836
- * @summary Checks if no tooltip modification when no KeyStroke modifier
- * @author Konstantin Eremin
- * @run applet/manual=yesno Test4887836.html
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary Checks for white area under the JColorChooser Swatch tab
+ * @run main/manual Test4887836
  */
 
-import java.awt.Color;
-import java.awt.Font;
-import javax.swing.JApplet;
-import javax.swing.JColorChooser;
-import javax.swing.UIManager;
+public class Test4887836 {
+
+    public static void main(String[] args) throws Exception {
+        String instructions = """
+                                If you do not see white area under the \"Swatches\" tab,\n +
+                                then test passed, otherwise it failed.""";
+
+        PassFailJFrame.builder()
+                .title("Test4759306")
+                .instructions(instructions)
+                .rows(5)
+                .columns(40)
+                .testTimeOut(10)
+                .testUI(Test4887836::createColorChooser)
+                .build()
+                .awaitAndCheck();
+    }
+
+    private static JColorChooser createColorChooser() {
+        JColorChooser chooser = new JColorChooser(Color.LIGHT_GRAY);
 
-public class Test4887836 extends JApplet {
-    public void init() {
-        UIManager.put("Label.font", new Font("Perpetua", 0, 36)); // NON-NLS: property and font names
-        add(new JColorChooser(Color.LIGHT_GRAY));
+        // NON-NLS: property and font names
+        UIManager.put("Label.font", new Font("Perpetua", 0, 36));
```

Comment:
Sorry, not related to the applet conversion but Perpetua ?? Pointless.  Won't exist.
Change it to Font.DIALOG. I think you should also remove the comment line.